utils/emoji_map.ts

```diff
@@ -66,7 +66,7 @@ export default class EmojiMap {
                     return {value: [emoji.name, emoji]};
                 }
 
-                return {done: true};
+                return {done: true, value: []};
```

Comment:
why do we need this?

Reply:
It needs an iterator,
error comes up ***Type 'undefined' must have a '[Symbol.iterator]()' method that returns an iterator***

Reply:
This will be fixed in https://mattermost.atlassian.net/browse/MM-49371